packages/flutter_tools/lib/src/android/application_package.dart

```diff
@@ -286,6 +335,12 @@ class _Attribute extends _Entry {
 class ApkManifestData {
   ApkManifestData._(this._data);
 
+  static String? _getActivityName(_Element element) {
+    final _Attribute? nameAttribute = element.firstAttribute('android:name');
+    // "io.flutter.examples.hello_world.MainActivity"
```

Comment:
Following up on [an earlier thread](https://github.com/flutter/flutter/pull/146548#discussion_r1613846849)—I now understand that this is called when parsing an XML dump, not the original XML.

```suggestion
    // "io.flutter.examples.hello_world.MainActivity" (Raw: "io.flutter.examples.hello_world.MainActivity")
```